### PR TITLE
remove password policy from settings

### PR DESCRIPTION
**Pull request by mxyng**
## Summary

use a static password policy which satisfies NIST guidelines. for more complex password policies, the recommendation is to use an external identity provider

### CODE REVIEW

File: internal/server/models/settings.go [hunk 1]
```diff
@@ -1,39 +1,9 @@
...
 
-import (
-	"github.com/infrahq/infra/api"
-)
-
 type Settings struct {
 	Model
 	OrganizationMember
 
 	PrivateJWK EncryptedAtRest
```
Comment (BruceMacD) [inline internal/server/models/settings.go:7]: Doesn't have to be part of this PR, but maybe with should move these settings to the organization entity.

**Review by BruceMacD — APPROVED** [review]

**Review by dnephin — APPROVED** [review]
Sorry for the conflicts. They should be easy to resolve, I think they are in deleted functions.

**Comment by mxyng** [comment]
> Sorry for the conflicts. They should be easy to resolve, I think they are in deleted functions.

All good. Conflicts were minimal. The only part that tripped me up slightly was `getPublicJWK` getting renamed and moved.

File: internal/server/models/settings.go (hunk 1 above)
Comment (mxyng) [reply internal/server/models/settings.go:7]: This will be done in #4045

**Pull request merged**